packages/gestalt/src/utils/KeyCodes.js

```diff
@@ -0,0 +1,4 @@
+// @flow strict
```

Comment:
@themindfuldev could you change this file name to `src/keyCodes.js`? We always want to avoid `utils` directories since they become a dumping ground for random things.

Reply:
Totally!